internal/connect/product.go

```diff
@@ -5,13 +5,19 @@ import (
 	"strings"
 )
 
-// Product represents an installed product or product information from API
-type Product struct {
+// BasicProduct represents basic information on product, mostly for API requests
+type BasicProduct struct {
 	Name    string `xml:"name,attr" json:"identifier"`
 	Version string `xml:"version,attr" json:"version"`
 	Arch    string `xml:"arch,attr" json:"arch"`
+}
+
+// Product represents an installed product or product information from API
+type Product struct {
+	BasicProduct
+	Release string `xml:"release,attr" json:"-"`
 	Summary string `xml:"summary,attr" json:"-"`
-	IsBase  bool   `xml:"isbase,attr" json:"-"`
+	IsBase  bool   `xml:"isbase,attr" json:"base"`
```

Comment:
was changing the json tag intentional?

Reply:
it was but not for this PR :o)
removed.